### PR TITLE
Additional fixes for vcpkg installation

### PR DESCRIPTION
**Pull request by ewertons**

### CODE REVIEW

File: build/.vsts-ci.yml [hunk 1]
```diff
@@ -264,6 +264,41 @@ phases:
...
+  steps:
+  - script: |
+     if [ -f "jenkins/linux_install_deps.sh" ]
+     then
+      sudo chmod 755 jenkins/linux_install_deps.sh
+      sudo -E ./jenkins/linux_install_deps.sh
+      sudo jenkins/linux_install_deps.sh
+     fi
+    displayName: 'build'
+    env:
```
Comment (massand) [inline build/.vsts-ci.yml]: Do we need these environment variables if we are not running the test?

Comment (ewertons) [reply build/.vsts-ci.yml]: Probably not. Shall remove them in another PR so we can scan the whole yaml file for jobs to clean

File: CMakeLists.txt [hunk 2]
```diff
@@ -50,10 +49,23 @@ if(NOT ${use_installed_dependencies})
...
         add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/deps/azure-c-shared-utility)
     endif()
+else()
+    if (NOT azure_macro_utils_cFOUND)
+        find_package(azure_macro_utils_c REQUIRED CONFIG)
+    endif ()
+    if (NOT umock_cFOUND)
+        find_package(umock_c REQUIRED CONFIG)
+    endif ()
+    if (NOT azure_c_shared_utility_FOUND)
```
Comment (YoDaMa) [inline CMakeLists.txt]: why is this different from the rest of the variables of this format?

Comment (ewertons) [reply CMakeLists.txt]: you mean the trailing _ should be removed. Thanks, that's correct

**Pull request merged**